xchange-stream-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingRawService.java

```diff
@@ -183,6 +184,13 @@ protected void handleMessage(JsonNode message) {
             }
             break;
           case PING:
+            // Sometimes the timestamp sent by Cex IO is up to a second ahead of current time. Then our pong sends a
+            // timestamp before Cex IO's own time.
```

Comment:
Where's this timestamp? Part of the websocket frame outside the message? It's certainly not in `CexioWebSocketPongMessage`.